### PR TITLE
Week1b rock paper scissors

### PR DESCRIPTION
**Pull request by copelandhouse2**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 01week/rockPaperScissors.js [hunk 1]
```diff
@@ -7,12 +7,51 @@ const rl = readline.createInterface({
...
-
+// Ah Rock Paper scissors
+// Ties: rock:rock; paper:paper; scissors:scissors
+// Hand1 wins: rock:scissors; paper:rock; scissors:paper
+// Hand2 wins: scissors:rock; rock:paper; paper:scissors
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
```
Comment (pieroapretto) [inline 01week/rockPaperScissors.js]: Nice data scrubbing!

File: 01week/rockPaperScissors.js [hunk 2]
```diff
@@ -7,12 +7,51 @@ const rl = readline.createInterface({
...
+      winningMessage = "Hand one wins!";
+    }
+  } // else if hand 1 = rock
+  else if (hand1 === 'paper') {
+    if (hand2 === 'rock') {
+      winningMessage = "Hand one wins!";
+    } else {
+      winningMessage = "Hand two wins!";
+    }
+  } // else if hand 1 = paper
```
Comment (pieroapretto) [inline 01week/rockPaperScissors.js]: Try to write your comments in plain English.

Ex: (Line 44) - Use cases if hand1 equals 'paper'

Comment (copelandhouse2) [reply 01week/rockPaperScissors.js]: Piero, just a note that I have updated my comments in this code to read English.  Hoping to raise my  grade.  See below commit.
